spec/models/geographical_area_spec.rb

```diff
@@ -49,4 +49,25 @@
       it { is_expected.not_to be_erga_omnes }
     end
   end
+
+  describe '#eu_member?' do
+    let(:geographical_area) { build(:geographical_area, id: geographical_area_id) }
```

Comment:
```suggestion
    subject(:geographical_area) { build(:geographical_area, id: geographical_area_id) }
```

Reply:
that is just the geographical_area and not the subject under test, `.eu_member?` is 

I tried it, but we need to stub the eu_members_ids before calling the eu_member.
(Alternatively, we should use VRC, which is overkill.)

Reply:
The truth is the subject is not usually a verb (a method call), it's something you interface and send messages too (a noun). We tighten up our tests a bunch by treating the subject as a verb and using `is_expected.to eq('foo')`. Most of the time a subject is either the class or the instance of that class as configured by the constructor.

I'd lean on just using VCR since it's easier/doesn't requiring the subject

Reply:
It's also really quick to do, I've used the `model#id` pattern in the past for the vcr cassette name (`geographical_area#it`)

Reply:
https://github.com/trade-tariff/trade-tariff-frontend/blob/5caceda749c2304865a3cf744a47790fb7dc57cd/spec/controllers/geographical_areas_controller_spec.rb#L4

Reply:
I've actually already loaded the cassette for the EU, even. So all you need to do is reuse that, here